buildrunner/__init__.py

```diff
@@ -686,6 +690,21 @@ def get_cache_archive_file(self, cache_name, project_name=""):
             os.makedirs(cache_dir)
         return local_cache_archive_file
 
+    @staticmethod
+    def clean_cache(log_func=print):
+        """
+        Clean cache dir
+        """
+        cache_dir = os.path.expanduser(
+            DEFAULT_CACHES_ROOT
```

Comment:
You probably want clean the directory listed in the config and to be safe it may not hurt to clean `DEFAULT_CACHES_ROOT` also.

https://github.com/adobe/buildrunner/blob/main/buildrunner/__init__.py#L672

The stand alone command currently doesn't guarantee to clean up all the caches that could have ever been written on the server. Currently there isn't a way for buildrunner to know which locations it has ever written caches to since different configs could in theory write to different locations. Perhaps we should discuss ways in a future task to make this clean caches command (mostly) ensures it clears any cache it ever wrote to.

Reply:
This suggests the `buildrunner` application would store a list of cache locations somewhere - we should discuss potential solutions to this.